Ccount.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
```

Comment:
Include comments! And next time, when you're so stuck, consult piazza. Questions can be asked anonymously. I suggest you review this assignment thoroughly so you actually understand these concepts.